python/ray/serve/built_application.py

```diff
@@ -76,3 +77,25 @@ def ingress(self) -> Optional[Deployment]:
                     return None
 
         return ingress
+
+    @property
+    def deploy_args_list(self) -> List[Dict]:
```

Comment:
`BuiltApplication` is marked as a public API. To avoid maintaining this utility as a public API, can you make it a separate utility or a private method?

Reply:
I've separated it out into a utility function!